test/config/seg_report.yaml

```diff
@@ -13,11 +13,12 @@ report: !Experiment
       final_transducer: !BiLSTMSeqTransducer {}
       policy_learning: !PolicyGradient 
         z_normalization: True
-        sample: 2
   train: !SimpleTrainingRegimen
     run_for_epochs: 1
     src_file: examples/data/head.ja
     trg_file: examples/data/head.en
+    loss_calculator: !MLELoss
+      repeat: 2
```

Comment:
Maybe we can set the default value of the repeat to be exp_global.enc_sample_num? This flag will bind together the number of sample in the encoder and in repeat. So we don't need to specify repeat manually.

Reply:
Or maybe we can get rid of `enc_global.enc_sample_num`?

Reply:
I see. I think we can just stack the sample on the fly while looping. OK, I think if we can just pass the unaccumulated loss generated from the repeat, everything should be settled.